packages/eas-cli/src/project/projectUtils.ts

```diff
@@ -111,7 +110,10 @@ export async function setProjectIdAsync(projectDir: string): Promise<ExpoConfig
   return (result.config as AppJSONConfig)?.expo;
 }
 
-export async function getProjectIdAsync(exp: ExpoConfig): Promise<string> {
+export async function getProjectIdAsync(
+  exp: ExpoConfig,
+  options: { env?: Env } = {}
```

Comment:
?
you are not using that field

Reply:
https://github.com/expo/eas-cli/pull/532/files#diff-e6ddd74b196c39db9302b9ec339e5eed15a1d6675d6b39f06898438e29feac1dR136